rules/rules-reviewed/eap8/eap7/jakarta-cdi.windup.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0"?>
+<ruleset xmlns="http://windup.jboss.org/schema/jboss-ruleset" id="jakarta-cdi"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+    <metadata>
+        <description>
+            This ruleset provides analysis of Java EE applications that need to change certain CDI-related method calls.
+        </description>
+        <dependencies>
+            <addon id="org.jboss.windup.rules,windup-rules-javaee,3.0.0.Final" />
+            <addon id="org.jboss.windup.rules,windup-rules-java,3.0.0.Final" />
+        </dependencies>
+        <sourceTechnology id="eap" versionRange="[7,8)" />
+        <targetTechnology id="eap" versionRange="[8,9)" />
+        <tag>JakartaEE</tag>
```

Comment:
I've added an `eap8` tag in https://github.com/windup/windup-rulesets/pull/770, maybe we should add that here too?

Reply:
Thanks Mark, done